lib/options.js

```diff
@@ -37,10 +38,10 @@ module.exports = {
       version: getVersion(),
       external: config.external || config.options.external || {},
       yuidoc: {
-        linkNatives: true,
-        quiet: true,
-        parseOnly: false,
-        lint: false,
+        linkNatives: yuidocOptions.linkNatives || true,
```

Comment:
Should be addressed (along with an error condition if `yuidocOptions` is `undefined`) by https://github.com/cibernox/ember-cli-yuidoc/pull/23.